project/userauth/tests.py

```diff
@@ -10,18 +12,11 @@
 
 class UserauthTests(APITestCase):
 
-    fixtures = ['users']
-
     def setUp(self):
-        # create a new user
-        model = get_user_model()
-        self.person = model.objects.create_user(
-            username='PetuniaPig',
-            email='pretty.piglet@pigfarm.org',
-            password='codebuddies'
+        self.user = UserFactory(
+            password=PostGenerationMethodCall('set_password', 'codebuddies')
         )
```

Comment:
The reason I'm doing this rather than

```py
self.user = UserFactory(password='codebuddies')
```

is because we need the value to be hashed.